torchdata/datapipes/iter/load/online.py

```diff
@@ -129,10 +137,11 @@ def __len__(self) -> int:
         return len(self.source_datapipe)
 
 
+@functional_datapipe("read_urls")
```

Comment:
I am not sure about this one. I don't have a good suggestion. `read_from_urls`?

Reply:
Or `read_from_remote`?

Reply:
Yeah. This was the part made me choosing pattern of `read_xxx` rather than `read_from_xxx`.
`read_from_remote` works for me.